model/src/main/java/uk/gov/hmcts/payment/api/model/govpay/GovPayCardPaymentService.java

```diff
@@ -59,6 +61,11 @@ public void refund(@NonNull String id, int amount, int refundAmountAvailable) {
         govPayClient.refundPayment(keyForCurrentService(), hrefFor(payment.getLinks().getRefunds()), new RefundPaymentRequest(amount, refundAmountAvailable));
     }
 
+    @Override
+    public List<GovPayPayment> search(Date startDate, Date endDate) {
+        return null;
```

Comment:
How about throwing MethodNot Supported ?

Reply:
This method has nothing to do in search, so it is left unimplemented.